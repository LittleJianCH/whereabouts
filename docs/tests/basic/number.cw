query (x, y, z) {
  equation x = 100
  equation y = -100
  equation z = 3.14
}
