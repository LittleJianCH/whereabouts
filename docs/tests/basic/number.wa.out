{ "success": true, "count": 1, "solutions": [{ "x": 100, "y": -100, "z": 3.14 }] }
